tests/integration/test_notes.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+from conftest import is_valid
+from models.notes import NotesModel
+
+
+@pytest.mark.usefixtures("client_class", "empty_test_db")
+class TestCreate:
+    def setup(self):
+        self.endpoint = "/api/tickets"
+        self.newNote = {"text": "We don't need no water"}
+
+    def test_creates_note_through_endpoint(
+        self, valid_header, create_admin_user, create_tenant, create_ticket
+    ):
+        create_admin_user()
+        create_tenant()
```

Comment:
Do we need these for the test?

Reply:
It appears we don't.  I didn't realize create_ticket includes adding admin and tenant.  Good catch!